gard_owl_ingest/mondo_mapping_status.py

```diff
@@ -242,21 +242,33 @@ def gard_mondo_mapping_status(
     outpath_mondo_sssom_curate = RELEASE_DIR / f'gard-mondo{file_suffix}_curation.sssom.tsv'
     write_tsv_with_comments(sssom_curate_df, MONDO_SSSOM_METADATA_PATH, outpath_mondo_sssom_curate)
 
-    # - gard-mondo.robot.template.sssom.tsv
+    # - mondo-gard.robot.template.sssom.tsv
     robot_df = sssom_df.drop(columns=['mondo_predicate_id'])\
-        .sort_values(by=['object_id', 'subject_id', 'proxy_id'])
-    robot_df['predicate_id'] = robot_df['predicate_id'].apply(
-        lambda x: x.replace('skos:exactMatch', 'MONDO:equivalentTo'))
+        .sort_values(by=['object_id', 'subject_id', 'proxy_id'])\
+        .rename(columns={
+            'subject_id': 'gard_id',
+            'object_id': 'mondo_id',
+            'proxy_id': 'source_external',
+            'predicate_id': 'source_type',
+            'object_label': 'mondo_label',
+        })
+    del robot_df['source_type']
+    del robot_df['mondo_label']
```

Comment:
Dropped `mondo_label` because it wasn't listed in the issue. Let me know if you want it back.

Reply:
No, its good you dropped it.